src/azul/indexer/__init__.py

```diff
@@ -375,7 +375,7 @@ def parse(cls, spec: str) -> Self:
         raise NotImplementedError
 
     @classmethod
-    def _parse(cls, spec: str) -> tuple[str, Prefix | None]:
+    def parse_prefix(cls, spec: str) -> tuple[str, Prefix | None]:
```

Comment:
The new name confuses me. Please consider alternatives and add description to docstring